providers/pyproject.toml

```diff
@@ -1,93 +0,0 @@
-# Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
There is no longer separate "providers" distribution. We only have individual provider's distributions.